README.md

```diff
@@ -11,6 +18,7 @@ Optional:
 - Kubernetes
 - Helm
 
+## Configuration steps (outside the scope of the repo)
```

Comment:
Parenthesis wasn't included in other README -- do you think we should inc. it across all?